src/Base/Tools.h

```diff
@@ -113,31 +113,54 @@ inline manipulator<int> blanks(int n)
 // ----------------------------------------------------------------------------
 
 template<class T>
+    requires std::is_arithmetic_v<T>
 inline T clamp(T num, T lower, T upper)
 {
-    return std::max<T>(std::min<T>(upper, num), lower);
+    return std::clamp<T>(num, lower, upper);

```

Comment:
@kadet1090 Heads up, in last push I swapped out the implementation of this function so it uses `std::clamp` instead:
https://en.cppreference.com/w/cpp/algorithm/clamp

We should probably deprecate `Base::clamp`, but that should be in a later PR instead

Reply:
nice!